cli/file_fetcher.rs

```diff
@@ -800,10 +801,12 @@ mod tests {
       assert_eq!(&(r4.media_type), &msg::MediaType::TypeScript);
       assert!(fs::read_to_string(&headers_file_name).is_err());
     });
+    drop(http_server_guard);
```

Comment:
Isn't the point of Drop that you don't need to call drop explicitly?

Reply:
rust compiler will make optimizations and drop early if you don't use a variable.